src/pages/Integrate.js

```diff
@@ -1,8 +1,7 @@
 import React, { useState } from "react";
 import { Flex, Box, Button, Text } from "@chakra-ui/core";
 import { GoLogoGithub } from "react-icons/go";
-import { DiMongodb } from "react-icons/di";
-import { SiDiscord, SiTrello, SiHeroku } from "react-icons/si";
+import { SiDiscord, SiHeroku, SiZoom, SiAsana } from "react-icons/si";
 
 const Integrate = () => {
   const [state, setState] = useState({
```

Comment:
I think we need to change this. Its hard to know what a, b, c, d, e is when reading the code